### DISCUSSION TITLE
Connecting RTSP Stream to AWS Kinesis

### DISCUSSION THREAD

**Discussion by Covert-Law-Enforcement** [AxisCommunications/discussions#229] [ACAP]
I am trying to develop an application to take the video stream from an Axis Camera and transfer it to AWS Kinesis. I have been looking for any sample code related to this and can't anything. If there are any samples that I could be referenced to that would be great!

**Comment by vivekatoffice** [comment]
Related link:
[axis-aws-kinesis-vsp](https://github.com/AxisCommunications/axis-aws-kinesis-vsp)
